homeassistant/components/group.py

```diff
@@ -112,6 +113,11 @@ def setup(hass, config):
 
 class Group(object):
```

Comment:
Let me see if I can fix it it tonight if I manage to do stay awake after
dinner this time.

On Wed, Apr 22, 2015, 18:25 Ryan Kraus <notifications@github.com> wrote:

> In homeassistant/components/group.py
> <https://github.com/balloob/home-assistant/pull/96#discussion_r28930146>:
>
> > @@ -112,6 +113,11 @@ def setup(hass, config):
> >
> >  class Group(object):
>
> I think mostly that would work. It would require a decent amount of
> re-factoring to the Group class though. As a future update, making the
> group and location entities use the Entity base class would be a good idea.
> It would definitely be a good practice to have all entities use the Entity
> base class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/96/files#r28930146>.
>